src/c++/perf_analyzer/genai-pa/genai_pa/llm_metrics.py

```diff
@@ -208,10 +242,11 @@ def _collect_llm_metrics(
             time_to_first_tokens.append(res_timestamps[0] - req_timestamp)
 
             # output token throughput
-            output_tokens = tokenizer(res_outputs)["input_ids"]
+            output_tokens = self._tokenize_response_outputs(res_outputs)
```

Comment:
Have we done any sanity testing on this? I think that the openai packet returns the number of tokens used, so we can use that to figure out if we're doing it right?

Reply:
Just checked their API documentation and openai only returns "usage" field (that includes token information) in the output only in the non-stream case. In streaming case, they do not report that information (for both legacy completion and chat completion).

Reply:
Please create a follow-up ticket just so we have something to level set on